src/app/offchain.js

```diff
@@ -0,0 +1,24 @@
+// @flow
+
+import { createApolloFetch } from 'apollo-fetch'
+
+import { ACCOUNT_KEY } from './account/actions'
+
+export const fetchAPI = createApolloFetch({
+  uri: 'http://localhost:3001/graphql',
```

Comment:
We need a way to use https://polymath-offchain.herokuapp.com in production but a different URL for development. Any ideas @bshevchenko ?

Reply:
process.env.NODE_ENV

Reply:
I wasn't aware that process.env works in React. That's very convenient! It looks like we can also make the URL configurable by giving it a REACT_APP_... environment variable as described at https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/template/README.md#adding-custom-environment-variables